collector/mongod/top_status.go

```diff
@@ -0,0 +1,41 @@
+package mongod
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/log"
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+// TopStatus represents top metrics
+type TopStatus struct {
+	TopStats TopStatsMap `bson:"totals,omitempty"`
+}
+
+// GetTopStats fetches top stats
+func GetTopStats(session *mgo.Session) (*TopStatus, error) {
+	results := &TopStatus{}
+	err := session.DB("admin").Run(bson.D{{"top", 1}}, &results)
```

Comment:
github.com/percona/mongodb_exporter/vendor/gopkg.in/mgo.v2/bson.DocElem composite literal uses unkeyed fields